temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,87 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 client certificate chain in PEM
+     *     format.
+     * @param keyPassword - the password of the keyFile, or null if it's not password-protected.
+     * @param keyInputStream - an input stream for a PKCS#8 client private key in PEM format.
+     * @param fingerprints - a list of SHA1 fingerprints in hexadecimal form of the server's root
+     *     certificate.
+     * @throws SSLException - when it was unable to build the context.
+     */
+    public Builder setCloudSslContext(
+        InputStream keyCertChainInputStream,
+        String keyPassword,
```

Comment:
FYI you probably want to use `char[]` rather than `String` for passing plaintext secrets around in memory: https://www.baeldung.com/java-storing-passwords

Not hugely important but should be a fairly easy change

Reply:
We need to pass this as a string to the key manager anyways, so it's not worth it to require char[] as it will get translated to the string anyways.